modules/videoio/test/test_camera.cpp

```diff
@@ -104,5 +104,42 @@ TEST(DISABLED_videoio_camera, v4l_read_framesize)
 
     capture.release();
 }
+TEST(COUNT_OF_CAMERA_FRAMES, v4l_poll)
+{
+        VideoCapture cap1(0);
+        VideoCapture cap2(2);
+
+        std::vector<VideoCapture> VCM;
+
+        VCM.push_back(cap1);
+        VCM.push_back(cap2);
+
+        std::vector<int> State;
+
+        std::vector<int> CountOfStates(2, 0);
+
+        int iteration = 500;
+        for(int i = 0; i < iteration; ++i)
+        {
+            cv::VideoCapture::waitAny(VCM, State, 50);
+            for(unsigned int j = 0; j < State.size(); ++j)
```

Comment:
I think it is better to use `VCM.size()` everywhere and add a check that `State.size()` is equal to it:
```.cpp
waitAny(...);
EXPECT_EQ(VCM.size(), State.size());
for (...; j < VCM.size(); ...)
```